### DISCUSSION TITLE
Allow for specifying the library import strings in vkInitialize

### DISCUSSION THREAD

**Discussion by jurganson** [amerkoleci/Vortice.Vulkan#43] [Ideas]
I have to change the hard-coded import paths in the `vkInitialize` method for certain platforms, but to do so I have to build my own binaries and change the paths in the code manually.

It would be awesome to be able to change the paths used by `vkInitialize` (parameters for `dlopen` and `LoadLibrary`) with some optional parameters or similar etc.

**Comment by amerkoleci** [comment]
Version: https://www.nuget.org/packages/Vortice.Vulkan/1.7.4 supports vkInitialize with optional libraryName parameter.

Cheers!

**Reply by jurganson** [reply]
Wooh just what I needed! Thank you @amerkoleci for the very fast feature response 🚀
